code/game/objects/items/devices/instruments.dm

```diff
@@ -268,3 +268,40 @@
 	item_state = "electronic"
 	lefthand_file = 'icons/mob/inhands/misc/devices_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/misc/devices_righthand.dmi'
+
+/obj/item/musicbeacon
+	name = "express delivery beacon"
+	desc = "Summon your tool of art."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "gangtool-red"
+	item_state = "radio"
+	var/static/list/display_names = list()
+
+/obj/item/musicbeacon/attack_self(mob/user)
+	if(!user.canUseTopic(src, BE_CLOSE, FALSE, NO_TK))
+		beacon_music(user)
+
+/obj/item/musicbeacon/proc/beacon_music(mob/M)
+	if(!display_names)
+		var/static/list/instruments
+		instruments = list(

```

Comment:
There's no reason for this to be on a second line and this just undermines the point of having it be static